scout/server/blueprints/alignviewers/controllers.py

```diff
@@ -338,16 +342,20 @@ def set_sample_tracks(display_obj: dict, case_groups: list, chromosome: str):
 
 
 def set_case_specific_tracks(display_obj, case_obj):
-    """Set up tracks from files that might be present or not at the case level
+    """Set up tracks from files that might be present for the focus case samples,
+        not fetched for all samples in the case group.
```

Comment:
Or actually just not for every sample in the case, which is a situation we sometime have for the other tracks as well, and have since handled. Even if there is only one UPD track for the case, it will have been computed for the proband, and marked and loaded as such.